### PR TITLE
Reviewing and editing Open Access 

### PR DESCRIPTION
**Pull request by vhellon**
### Summary

Editing the chapter, moving sections around, adding references

### What should a reviewer concentrate their feedback on?

- [ ] Check the new flow makes sense
- [ ] Do we think it should be written as 'Open Access' or 'open access' throughout?
- [ ] Not completely happy with the future of OA section- any additions/edits welcome
- [ ] Everything looks ok?

### Acknowledging contributors

- [] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.

### CODE REVIEW

File: book/website/reproducible-research/open/open-access.md [hunk 1]
```diff
@@ -4,32 +4,73 @@
...
-This movement took the name of "open access" [{term}`def<Open Access>`] and established two initial strategies to achieve its final goal: self-archiving and open access publishing.
+At the turn of the 21st century, the Open Access [{term}`def<Open Access>`] movement was established, principally through the Budapest Open Access Initative and the Berlin Declaration on Open Access. 
+A quote from the Budapest Open Access Initative sums up the objective:
+>"We mean its free availability on the public internet, permitting any users to read, download, copy, distribute, print, search, or link to the full texts of these articles...the only constraint on reproduction and distribution, and the only role for copyright in this domain, should be to give authors control over the integrity of their work and the right to be properly acknowledged and cited"
+
+It has been estimated around 28% of the scholarly literature is Open Access {cite}`Piwowar2018stateofOA`and in 2020 more outputs were published through Open Access channels than traditional subscription channels globally {cite}`Dimensions2021openaccesssurpasses`.
+
+(rr-open-access-routes)=
+## Routes for Open Access
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:19]: Might benefit from a short paragraph here acknowledging there's lots of different ways to achieve open access depending on your resources, the norms in your field or discipline, and funder restrictions

File: book/website/reproducible-research/open/open-access.md [hunk 2]
```diff
@@ -4,32 +4,73 @@
...
+(rr-open-access-preprints)=
+### Preprints
+
+A preprint is a scientific manuscript that is uploaded by the authors to a public server before peer review. 
+The first preprint server called arXiv was established in 1991 for physical sciences. 
+Likely due to the inefficiency of traditonal journals, preprint servers for other disciplines have grown hugely in popularity over the last 5 years, with the emergence of bioRxiv, medRxiv, ChemRxiv, SocArXiv and many more.
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:25]: We should link to these repositories!

File: book/website/reproducible-research/open/open-access.md [hunk 3]
```diff
@@ -4,32 +4,73 @@
...
+Some journals will not allow a paper to be submitted if its already been posted on a preprint server as they consider it 'prior publication', however the ongoing popularity of preprints has forced many publishers to embrace preprints, or in the case of journals like eLife, actively encourage them {cite}`elife2020preprints`.
+You can find out more about preprints at [ASAPbio](https://asapbio.org/preprint-info).
+
+(rr-open-access-journal-publishing)=
+### Open Access Journal publishing
+
+Traditionally many journals are subscription journals, meaning their content is behind a paywall. 
+Universities may have subscriptions to that journal, allowing you to access its content if you are based at a subscribing institution or you may be able to pay a one off fee to access an individual article.
+
+The Open Access movement led to the creation of Open Access journals, where the content is open for anyone to access.
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:36]: Suggest: "where the content is accessible for everyone, with no need to pay, or be a member of a subscribing institution."

File: book/website/reproducible-research/open/open-access.md [hunk 4]
```diff
@@ -4,32 +4,73 @@
...
+Traditionally many journals are subscription journals, meaning their content is behind a paywall. 
+Universities may have subscriptions to that journal, allowing you to access its content if you are based at a subscribing institution or you may be able to pay a one off fee to access an individual article.
+
+The Open Access movement led to the creation of Open Access journals, where the content is open for anyone to access. 
+The number of these journals is still increasing rapidly, with 17,000 listed in the Directory of Open Access Journals [DOAJ](http://www.doaj.org) (data fetched November 2021). 
+
+(rr-open-access-gold-publishing)=
+#### Gold Open Access publishing
+
+Gold Open Access Publishing (or sometimes referred to as libre) can be defined as paying an Article Processing Charge (APC) to a journal so they publish the final version of your article, which is then permanently and freely available online for anyone.
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:42]: "publish the final version of your article **under an open access license**, which is then...."

File: book/website/reproducible-research/open/open-access.md [hunk 5]
```diff
@@ -4,32 +4,73 @@
...
+The author will retain copyright of their article, usually via a Creative Commons licence of their choice, which dictates what others can do with the article.
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:43]: Link Creative Commons website here - I think it's the first time it's mentioned

File: book/website/reproducible-research/open/open-access.md [hunk 6]
```diff
@@ -4,32 +4,73 @@
...
+You can find out more about APCs in this [video](https://www.youtube.com/watch?v=QAeTKUpZq_Y).
+
+There can often be misconceptions around publishing in Open Access journals- as they are newer to the system they are often wrongly thought of as 'less rigorous' or 'lower quality' than the 'traditional' journals. 
+Some researchers have also been caught up in what is referred to as 'predatory publishing'- where they are tricked into paying an article processing charge to journals that are fradulent and do not provide editorial processes that check articles for quality and legitimacy.
+Following the [Think.Check.Submit guidelines](https://thinkchecksubmit.org/) can help researchers to avoid predatory journals.
 
-(rr-open-access-whatis-repositories)=
-### Repositories and Self-Archiving
+A criticism around gold Open Access publishing is also the cost. 
+Article Processing Charges (APCs) can generally be around $2000 or in some cases more, which can therefore be prohibitive for authors across the globe.
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:51]: We've already defined APC above so you can just use APC from now on :)

File: book/website/reproducible-research/open/open-access.md [hunk 7]
```diff
@@ -45,59 +86,37 @@ Finally, some journals do allow researchers to deposit the final published versi
...
-For instance, the accepted version could be made publicly available after publication, but the published version must wait 12 months.
-
-(rr-open-access-whatis-publishing)=
-### Open Access Publishing
+For instance, the accepted version could be made publicly available after publication, but the published version must wait 12 months. You can check journal policies on self-archiving using [SHERPA/RoMEO](http://www.sherpa.ac.uk/romeo/index.php).
 
-Open access publishing attempts to ensure permanent open access to all the articles published in journals and results in the creation of the open access journals.
-The number of open access journals has increased during the last years, according to the Directory of Open Access Journals \([DOAJ](http://www.doaj.org)\). Currently, there are more than 12,000.
-Open access journals must provide free access to its contents, but it also must licence them to allow reusability.
+An overview of Open Access and all the routes mentioned above are covered in these videos from [Welch Medical Library](https://www.youtube.com/watch?v=CFa2QeMgk9k) and the  [Unpaywall team](http://go.sirsidynix.com/2020-04-29-Episode-1-Introduction-to-Open-Access.html).
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:91]: double space before the Unpaywall

File: book/website/reproducible-research/open/open-access.md [hunk 8]
```diff
@@ -45,59 +86,37 @@ Finally, some journals do allow researchers to deposit the final published versi
...
-A restriction could be an embargo period, or limits on the allowed version to be deposited in such archives.
-In doing this, it is crucial to make sure you are aware of the copyright implications of any documents/agreements you make when submitting your manuscript to a journal.
-If your institution does not have an institutional repository, advocate for the creation of one.
-You can check journal policies on self-archiving using [SHERPA/RoMEO](http://www.sherpa.ac.uk/romeo/index.php).
+The push for open is prevalent across the globe- [SciELO](https://en.wikipedia.org/wiki/SciELO) was created to meet the scientific communication needs of developing countries and provides an efficient way to increase visibility and access to scientific literature.
 
-(rr-open-access-practice-publication)=
-### Publication
+Funders have also taken acion by becoming publishers themselves (referred to as Diamond Open Access). Platforms such as [Open Research Europe](https://open-research-europe.ec.europa.eu/) and [Wellcome Open Research](https://wellcomeopenresearch.org/) are Open Access publishing platforms for researchers funded by that specific funder.
```
Comment (Arielle-Bennett) [inline book/website/reproducible-research/open/open-access.md:118]: "acion" - typo 😄

File: book/website/_bibliography/references.bib [hunk 9]
```diff
@@ -1009,3 +1009,43 @@ @misc{Panter2021bestpractices
...
+	volume = {5},
+	pages = {632},
+	year = {2016},
+	month = {Sep},
+	publisher = {F1000Research},
+	doi = {10.12688/f1000research.8460.3}
+}
+
+@article{Piwowar2018stateofOA,
+	author = {Piwowar, H., Priem, J., Lariviere, V., Alperin, J.P., Matthias, L., Norlander, B., Farley, A., West, J. and Haustein, S},
```
Comment (c-martinez) [inline book/website/_bibliography/references.bib:1026]:
```suggestion
	author = {Piwowar, H. and Priem, J. and Lariviere, V. and Alperin, J.P. and Matthias, L. and Norlander, B. and Farley, A. and West, J. and Haustein, S},
```

File: book/website/reproducible-research/open/open-access.md [hunk 10]
```diff
@@ -4,32 +4,73 @@
...
+The first scientific journal, Philosophical Transactions, was established in 1665 and published letters about scientific observations and experimentations.
+The 'commercialisation' of scientific publishing began in the 1940s and for many years publications, such as through a journal, conference proceedings or book were available to the public if purchased through a subscription fee or individually.
+
 However, new knowledge is built by synthesizing current scholarship and then building upon it.
-At the turn of the 21st century, a new movement appeared with a clear objective: make all the research results available to anyone interested in reading it, free of charge by any user, with no technical obstacles such as mandatory registration or login to specific platforms.
-This movement took the name of "open access" [{term}`def<Open Access>`] and established two initial strategies to achieve its final goal: self-archiving and open access publishing.
+At the turn of the 21st century, the Open Access [{term}`def<Open Access>`] movement was established, principally through the Budapest Open Access Initative and the Berlin Declaration on Open Access.
```
[the remark this answers is not in the document]
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

File: book/website/reproducible-research/open/open-access.md (hunk 1 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md:19]: Have done, let me know what you think?

File: book/website/reproducible-research/open/open-access.md (hunk 2 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

File: book/website/reproducible-research/open/open-access.md (hunk 3 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

File: book/website/reproducible-research/open/open-access.md (hunk 4 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

File: book/website/reproducible-research/open/open-access.md (hunk 5 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

File: book/website/reproducible-research/open/open-access.md (hunk 6 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

File: book/website/reproducible-research/open/open-access.md (hunk 7 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

File: book/website/reproducible-research/open/open-access.md (hunk 8 above)
Comment (vhellon) [reply book/website/reproducible-research/open/open-access.md]: done :)

**Comment by vhellon** [comment]
@EstherPlomp Not sure if you want to review or not but tagging you just in case!

File: book/website/reproducible-research/open/open-access.md [hunk 11]
```diff
@@ -4,32 +4,79 @@
 (rr-open-access-whatis)=
 ## What Is Open Access?
 
-One of the most common ways to disseminate research results is by writing a manuscript and publishing it in a journal, conference proceedings or book. For many years those publications were available to the public if purchased through a subscription fee or individually.
+Disseminating and sharing scientific results has been part of research since it was established.
+The first scientific journal, Philosophical Transactions, was established in 1665 and published letters about scientific observations and experimentations.
+The 'commercialisation' of scientific publishing began in the 1940s and for many years publications, such as through a journal, conference proceedings or book were available to the public if purchased through a subscription fee or individually.
```
Comment (EKaroune) [inline book/website/reproducible-research/open/open-access.md:9]:
```suggestion
The 'commercialisation' of scientific publishing began in the 1940s and for many years publications, such as through a journal, conference proceedings or book, were available to the public if purchased through a subscription fee or individually.
```

**Comment by malvikasharan** [comment]
@all-contributors please add @vhellon for review, content

**Review by malvikasharan — APPROVED** [review]
Looks great!

**Pull request merged**